Compiling errors

Hi, been trying to get this to compile to test out and write about the OS release.

Coming up against this on Linux:

> [  5%] Building CXX object CMakeFiles/good_robot.dir/InputManager.cpp.o
> In file included from /home/liam/Downloads/Good-Robot-master/InputManager.h:4,
>                  from /home/liam/Downloads/Good-Robot-master/InputManager.cpp:2:
> /home/liam/Downloads/Good-Robot-master/inputval.h:19:3: error: ‘SDL_Scancode’ does not name a type; did you mean ‘SDL_GrabMode’?
>    19 |   SDL_Scancode key_val, key_alt;
>       |   ^~~~~~~~~~~~
>       |   SDL_GrabMode
> make[2]: *** [CMakeFiles/good_robot.dir/build.make:148: CMakeFiles/good_robot.dir/InputManager.cpp.o] Error 1
> make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/good_robot.dir/all] Error 2
> make: *** [Makefile:104: all] Error 2

Also, could we get the compiling instructions added, not just the needed libs? Just to make it clearer.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,9 +28,11 @@ add_executable(good_robot WIN32 ${good_robot_SRC})
 add_definitions(-DCMAKE_BUILD)
 
 set(SDL_BUILDING_LIBRARY ON)
-find_package(SDL REQUIRED)
-target_link_libraries (good_robot ${SDL_LIBRARY})
-include_directories (${SDL_INCLUDE_DIR})
+# use pkg-config to find SDL2
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(SDL2 REQUIRED sdl2)
+target_link_libraries (good_robot ${SDL2_LIBRARIES})
+include_directories (${SDL2_INCLUDE_DIRS})
 
 find_package(OpenGL REQUIRED)
 target_link_libraries (good_robot ${OPENGL_LIBRARIES})

